pkg/controller/service/service_controller.go

```diff
@@ -306,13 +315,15 @@ func (s *ServiceController) createLoadBalancerIfNeeded(key string, service *v1.S
 		service.Status.LoadBalancer = *newState
 
 		if err := s.persistUpdate(service); err != nil {
-			return fmt.Errorf("Failed to persist updated status to apiserver, even after retries. Giving up: %v", err), notRetryable
+			return fmt.Errorf("Failed to update service %s/%s status to apiserver: %v",
```

Comment:
This error message seems a bit redundant as it will be encapsulated again. Could we somehow combine it with https://github.com/kubernetes/kubernetes/pull/53196/files#diff-1f7f903e25ab8bcbc514bb1e532e997eR301?